conf/pg_config.dist.yml

```diff
@@ -139,6 +139,12 @@ specialPGEnvironmentVars:
   # Size in pixels of dynamically-generated images, i.e. graphs.
   onTheFlyImageSize: 400
 
+  # To enable Rserve (the R statistical server), uncomment the following two
+  # lines. The R server needs to be installed and running in order for this to
+  # work. See http://webwork.maa.org/wiki/R_in_WeBWorK for more info.
+  #Rserve:
+  #  host: localhost
+
```

Comment:
It looks like this is needed for the standalone server only?   Either way, we should update the wiki for this.

Reply:
This is needed for anyone not using webwork2 to render PG problems.

The standalone renderer has its own `pg_config.yml` file.  This will need to be added to that file.

I don't think there is a wiki page where this is discussed.  Where in particular are you thinking this should be commented on in the wiki?